src/auth-service/config/constants.js

```diff
@@ -165,6 +165,7 @@ const defaultConfig = {
   DEVELOPERS_EMAILS: process.env.DEVELOPERS_EMAILS,
   PARTNERS_EMAILS: process.env.PARTNERS_EMAILS,
   YOUTUBE_CHANNEL: process.env.AIRQO_YOUTUBE,
+  CHAMPIONS_FORM: process.env.CHAMPIONS_FORM,
```

Comment:
Hi @BenjaminSsempala , could we just add the value here straight and not use secrets to minimise complexity since this value is not necessarily a "secret".

Reply:
Thanks @Baalmart I've modified this.